src/App/Hovedside/TjenesteBoksContainer/IAwebboks/IAwebboks.tsx

```diff
@@ -38,7 +38,7 @@ const IAwebboks = () => {
                 <span className={'legemeldt-sykefravær-prosent'}>
                     {sykefravær.prosent.toString()} %
                 </span>
-                <> legemeldt sykefravær i din {statistikktype(sykefravær.type)}. Slik kan du forebygge sykefravær.   </>
+                <> Legemeldt sykefravær i din {statistikktype(sykefravær.type)}. Slik kan du forebygge fravær.   </>
```

Comment:
Jeg tenker det skal være liten bokstav her ("legemeldt"), siden det er en fortsettelse på tallet foran (sykefravær-prosent).
```suggestion
                <> legemeldt sykefravær i din {statistikktype(sykefravær.type)}. Slik kan du forebygge fravær.   </>
```